enterprise/replay/replay.go

```diff
@@ -97,8 +97,8 @@ func (handle *Handler) generatorLoop(ctx context.Context) {
 			status := jobsdb.JobStatusT{
 				JobID:         job.JobID,
 				JobState:      jobsdb.Executing.State,
-				ExecTime:      time.Now(),
-				RetryTime:     time.Now(),
+				ExecTime:      time.Now().UTC(),
```

Comment:
To keep parity with the created_at times in jobs table, I had to convert all exec_times & retry_times to UTC.

created_at times in jobs table are assigned by postgres (because of default value) & postgres deals in UTCs.

Reply:
Do we also parse those dates as UTC when we read rows from the database too?

Reply:
1. True.
2. No
3. Don't think so

We can use `timestamp with time zone` too. But we need to change the way we create jobs table then.

Reply:
Indeed, we need to adapt our create table statements for jobs & job status tables

Reply:
done